seatunnel-connectors-v2/connector-file/connector-file-ftp/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/ftp/sink/FtpFileSinkFactory.java

```diff
@@ -57,6 +57,7 @@ public OptionRule optionRule() {
                         BaseSinkConfig.FILE_FORMAT_TYPE,
                         FileFormat.JSON,
                         BaseSinkConfig.TXT_COMPRESS)
+                .conditional(BaseSinkConfig.FILE_FORMAT_TYPE, FileFormat.EXCEL)
```

Comment:
Why add this?

Reply:
![image](https://user-images.githubusercontent.com/51053924/230291279-c4d8ce2d-f88a-4424-8829-1515db301f2b.png)

you need update it.